yarn-project/circuit-types/src/interfaces/block-builder.ts

```diff
@@ -9,10 +9,16 @@ export interface BlockBuilder extends ProcessedTxHandler {
   /**
    * Prepares to build a new block. Updates the L1 to L2 message tree.
    * @param numTxs - The complete size of the block.
+   * @param numTxsEffects - The total number of transaction effects in the block
```

Comment:
Ah I always thought each component was a single effect e.g. one note is one effect? So the number of effects would be the number of items, rather than the number of txs! If that's wrong I will change it to +fields

Reply:
Oh what I thought was we new a TxEffect for each tx, so one tx one txEffect 😂  But what you said also makes sense! Although here it's also not the number of items, if a tx has two notes, it doesn't add 2 to numTxsEffects, instead it adds 3 because there is also a field for prefix.

Reply:
True! When I wrote this, I didn't really plan for prefixes, so it makes sense to rename it. Something like `numBlobFields` or `numTxsBlobFields`?

Reply:
`numBlobFields` sounds good!